4.0/init.sh

```diff
@@ -5,22 +5,39 @@ OSMFILE=${PROJECT_DIR}/data.osm.pbf
 if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
   echo "Downloading Wikipedia importance dump"
   curl https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
+elif [ -f "$IMPORT_WIKIPEDIA" ]; then
+  # use local file if asked
+  ln -s "$IMPORT_WIKIPEDIA" ${PROJECT_DIR}/wikimedia-importance.sql.gz
 else
   echo "Skipping optional Wikipedia importance import"
 fi;
 
 if [ "$IMPORT_GB_POSTCODES" = "true" ]; then
   curl https://nominatim.org/data/gb_postcode_data.sql.gz -L -o ${PROJECT_DIR}/gb_postcode_data.sql.gz
+elif [ -f "$IMPORT_GB_POSTCODES" ]; then
+  # use local file if asked
+  ln -s "$IMPORT_GB_POSTCODES" ${PROJECT_DIR}/gb_postcode_data.sql.gz
 else \
   echo "Skipping optional GB postcode import"
 fi;
 
 if [ "$IMPORT_US_POSTCODES" = "true" ]; then
   curl https://nominatim.org/data/us_postcode_data.sql.gz -L -o ${PROJECT_DIR}/us_postcode_data.sql.gz
+elif [ -f "$IMPORT_US_POSTCODES" ]; then
+  # use local file if asked
+  ln -s "$IMPORT_US_POSTCODES" ${PROJECT_DIR}/us_postcode_data.sql.gz
 else
   echo "Skipping optional US postcode import"
 fi;
 
+if [ "$IMPORT_TIGER_ADDRESSES" = "true" ]; then
+  curl https://nominatim.org/data/tiger2021-nominatim-preprocessed.csv.tar.gz -L -o ${PROJECT_DIR}/tiger2021-nominatim-preprocessed.csv.tar.gz
```

Comment:
Can you please also delete this where the other dumps are deleted around line 102